vcx/libvcx/src/messages/agent_utils.rs

```diff
@@ -432,6 +462,46 @@ fn update_agent_info_v2(to_did: &str, com_method: ComMethod) -> VcxResult<()> {
     Ok(())
 }
 
+pub fn update_agent_webhook(webhook_url: &str) -> VcxResult<()> {
+    trace!("update_agent_webhook >>> webhook_url: {:?}", webhook_url);
+
+    let to_did = settings::get_config_value(settings::CONFIG_REMOTE_TO_SDK_DID)?;
+
+    let com_method: ComMethod = ComMethod {
+        id: String::from("123"),
```

Comment:
Was `123` previously used as id of update comm message as well?

Reply:
Oh okay now I see that in `pub extern fn vcx_agent_update_info` which is also building and sending `ComMethod` structure, it's using `id` passed by user in argument and I haven't found it documented.
@jovfer is it okay to set here such a dummy value like 123? Is `id` only relevant in case of `com_method.e_type == 1`?

Reply:
@ryMarsh44 @Artemkaaas could you take a look on this question?

Reply:
@jovfer I do not know for certain but as far as I can remember, the field `id` is not actually used in any of the agent implementations I've seen. I believe "123" is a default value.